formatters/coveragepy/coveragepy_test.go

```diff
@@ -26,4 +26,5 @@ func Test_Parse(t *testing.T) {
 	sf := f.SourceFiles[11]
 	r.Equal("codeclimate_test_reporter/components/runner.py", sf.Name)
 	r.InDelta(85.71, sf.CoveredPercent, 1)
+	r.Len(sf.Coverage, 82)
```

Comment:
Do you mind including two more assertions: one that asserts there's a `null` at an index we expect and another that asserts there's a non-`null` at an index we expect? Preferably using indexes near boundaries to try and catch off-by-one errors.